src/scenic/syntax/ast.py

```diff
@@ -1250,3 +1250,13 @@ def __init__(self, left: ast.AST, right: ast.AST, *args: any, **kwargs: any) ->
         self.left = left
         self.right = right
         self._fields = ["left", "right"]
+
+
+class IntersectsOp(AST):
+    __match_args__ = ("left", "right")
+
+    def __init__(self, left: ast.AST, right: ast.AST, *args: any, **kwargs: any) -> None:
```

Comment:
```suggestion
    def __init__(self, left: ast.AST, right: ast.AST, *args: Any, **kwargs: Any) -> None:
```

(I see this file uses `any` all over the place, but that's wrong: `any` is a built-in function, whereas we want `Any` from the `typing` module. We can make another PR to fix those, or you can find-and-replace them here.)

Reply:
Do we even need typing information here? If the type can be any then I feel like it's cleaner to just leave it blank and let any analysis automatically assume it can be any type.

Reply:
We don't use the typing information. I assume @shumbo put it in in case we later run a type checker on the implementation. I think the best practice is to put `Any` for arguments which are known to really be able to take on any type (or at least a sufficiently broad range of types that there's no easy way to specify them more precisely) rather than leaving the arguments unannotated: the latter can be mistaken for "we just haven't gotten around to annotating this yet".

Reply:
Sounds good, I've put it on the todo list!